themes/default/content/blog/troubleshooting-with-diagnostics/index.md

```diff
@@ -0,0 +1,148 @@
+
+## Fixing errors
+
+Sometimes, when exploring the diagnostic data, you can find out what's up pretty quickly by checking the last 10 or 20 lines. You always want to check a handful of lines backwards when going through this kind of information because the problem spots will often appear before child processes sending information and shutting down, other processes failing due to dependencies, etc. I often scan a bit up to something that appears familiar, such as a handful of lines I can find in a previous run, and then work my way down from there until I identify a problem spot.
+
+If it's not a problem in your code, however, it can be difficult to track down the error in question since there are so many moving parts. Based on the behavior of the program before it failed, you can track down which part you need to search for. If a provider is failing, for example, I'd check to find out whether the right versions of provider plugins are being installed, which requires searching the data for `plugins.go` or another similar process. If I'm running into an authentication problem, I'll check the `api.go` details and search for anything out of place.
+
+## Submitting bug reports
+
+Sometimes, though, you might not be able to find a clear reason why your program failed. Or you found a line that didn't make sense, and it really does seem like a bug. It happens! All of the detailed logs you were able to gather are useful for us to fix the problem, along with the details on what system you're trying to run on (e.g., Windows 10, a Mac with an M1 chip) and the versions of Pulumi, the provider, and the language host. We'd love to get a bug report if you find something that doesn't make sense, and if you can find the line in the code that you think might be the problem by [exploring the repo](https://github.com/pulumi/pulumi/), even better! We do like getting community pull requests, too (though please keep in mind that some of the codebase is automatically generated, so we might respond with a note to that effect). If you're not sure if it's a bug, you can [open a Discussion](https://github.com/pulumi/pulumi/discussions) or ask when you open the issue.
```

Comment:
>details on what system you're trying to run on 
Pretty much all the info we'd care about gets printed by "pulumi about". Maybe worth suggesting that to people?